src/test/kotlin/no/nav/soknad/arkivering/soknadsmottaker/rest/ReceiverTests.kt

```diff
@@ -35,13 +38,20 @@ class ReceiverTests {
 		assertEquals(topic, captor.value.topic(), "Should send to the right topic")
 		assertEquals(1, captor.value.headers().headers(MESSAGE_ID).count(), "Should have a MESSAGE_ID header")
 		assertEquals("BIL", captor.value.value().getArkivtema(), "Should have correct tema")
-		assertEquals(errorsBefore + 0.0, Metrics.mottattErrorGet("BIL"), "Should not cause errors")
-		assertEquals(sentInBefore + 1.0, Metrics.mottattSoknadGet("BIL"), "Should increase counter by 1")
+		assertEquals(getDouble(errorsBefore) + 0.0, metrics.mottattErrorGet("BIL"), "Should not cause errors")
+		assertEquals(getDouble(sentInBefore) + 1.0, metrics.mottattSoknadGet("BIL"), "Should increase counter by 1")
+
+		metrics.unregister()
+	}
+
+	private fun getDouble(d: Double?) : Double {
```

Comment:
Jeg vil anbefalle å bruke `!!`-operatoren istedenfor å ha en metod som denne. Man bör vare litt forsiktig med å bruke `!!`, men i tester som dette er det bare å ta i bruk. Så linie 41 for eksempel, kunde bli slik:
`assertEquals(errorsBefore!! + 0.0, metrics.mottattErrorGet("BIL")!!, "Should not cause errors")`

Reply:
Fikset